src/app/lib/index.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const path = require('path');
+
+const randomItem = (items) => items[Math.floor(Math.random() * items.length)];
```

Comment:
```suggestion
const getRandomItem = (items) => items[Math.floor(Math.random() * items.length)];
```
Methods are usually started with verb.